amgut/lib/data_access/ag_data_access.py

```diff
@@ -529,9 +552,10 @@ def ag_update_kit_password(self, kit_id, password):
         """
         password = bcrypt.encrypt(password)
 
-        self.get_cursor().callproc('ag_update_kit_password',
-                                   [kit_id, password])
-        self.connection.commit()
+        sql = """UPDATE AG_KIT
```

Comment:
Minor, but it seems like you've been writing the name of the tables with all lowercase, except here ...